ComponentTextKit/CKLabelComponent.h

```diff
@@ -69,4 +69,8 @@ struct CKLabelAttributes
 + (instancetype)newWithLabelAttributes:(const CKLabelAttributes &)attributes
                         viewAttributes:(const CKViewComponentAttributeValueMap &)viewAttributes;
 
++ (instancetype)newWithLabelAttributes:(const CKLabelAttributes &)attributes
+                        viewAttributes:(const CKViewComponentAttributeValueMap &)viewAttributes
+                                  size:(const CKComponentSize &)size;
```

Comment:
http://componentkit.org/docs/avoid-overrides.html

Replace the old ones rather than adding new ones.